Lib/test/test_gettext.py

```diff
@@ -213,14 +212,6 @@ def test_the_alternative_interface(self):
         # Try unicode return type
         t.install()
         eq(_('mullusk'), 'bacon')
-        # Test installation of other methods
-        import builtins
-        t.install(names=["gettext", "lgettext"])
```

Comment:
Keep the test, but use "ngettext" or "pgettext" instead of "lgettext".

It would be nice to test also that "install" in the names list does not set `builtins.install`.